test/forward_backward_compatibility/check_forward_backward_compatibility.py

```diff
@@ -162,9 +146,12 @@ def check_bc(existing_schemas):
     is_bc = True
     broken_ops = []
     for existing_schema in existing_schemas:
-        if allow_listed(existing_schema):
+        if allow_listed_with_deadline(existing_schema):
             print("schema: ", str(existing_schema), " found on allowlist, skipping")
             continue
+        if allow_listed_without_deadline(existing_schema):
+            print("schema: {} is found in forever BC breaking list. This is very dangerous".format(str(existing_schema)))
```

Comment:
Maybe rephrase it to be something like:
{} is in allowlist for BC-breaking evolution without deadline. This is dangerous, do not use unless you are sure there will not be downstream consequences.

Reply:
sounds good.

Reply:
We should be clear this is skipping BC and FC schema checks, but I think we would need to add more details about exactly how this is "dangerous" and when, if ever, it's OK to do this. If we're not sure then having someone contact us is probably the appropriate recommendation